src/app/tests/suites/DL_LockUnlock.yaml

```diff
@@ -270,6 +270,94 @@ tests:
       response:
           error: FAILURE
 
+    - label: "Try to unlock the door with valid PIN for the third time"
+      command: "UnlockDoor"
+      timedInteractionTimeoutMs: 10000
+      arguments:
+          values:
+              - name: "PINCode"
+                value: "123456"
+
+    - label: "Verify that lock state attribute value is set to Unlocked"
+      command: "readAttribute"
+      attribute: "LockState"
+      response:
+          value: 2
+
+    - label: "Try to unlock the door with invalid PIN for the first time"
+      command: "UnlockDoor"
```

Comment:
Don't we need to re-lock it first, before trying to unlock more?

Reply:
yes we should. my bad